Examples/Detectors/ContextualDetector/include/ActsExamples/ContextualDetector/AlignedDetector.hpp

```diff
@@ -30,17 +29,34 @@ class GenericDetectorElement;
 }  // namespace ActsExamples
 
 namespace ActsExamples::Contextual {
-class InternallyAlignedDetectorElement;
-class InternalAlignmentDecorator;
 
-class AlignedDetector {
+// New struct to represent a misalignment group
+struct MisalignmentGroup {
```

Comment:
Not sure I understand why we need this struct twice, it's very similar to the GroupedMisalignment right?